atomic_reactor/plugins/pre_add_labels_in_df.py

```diff
@@ -201,6 +190,59 @@ def add_as_an_alias(new, old, is_old_inherited):
             self.log.debug("applied only some aliases, following old labels were not found: %s",
                            ", ".join(not_applied))
 
+        def check_if_all_same(labels_to_check, source):
+            if labels_to_check:
+                label_value = source[next(iter(labels_to_check))]
```

Comment:
I think `check_if_all_same` should convert the argument to a list so that it would be easier to undestand that only the first value is used for comparison